cppcache/integration/test/resources/pr_app_client_cache.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<client-cache xmlns="http://geode.apache.org/schema/cache"
+			  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+			  xsi:schemaLocation="http://geode.apache.org/schema/cache http://geode.apache.org/schema/cache/cache-1.0.xsd"
```

Comment:
@pivotal-jbarrett shouldn't the schemaLocation for both of these be "http://geode.apache.org/schema/cpp-cache/cpp-cache-1.0.xsd"?

Reply:
No, it's lost of pairs of namespace uri to url mapping.

Reply:
Sadly I am not really aware with the XML internals, but just to be clear, should I change the namespace to: https://geode.apache.org/schema/cpp-cache and the schema to https://geode.apache.org/schema/cpp-cache/cpp-cache-1.0.xsd ?

Reply:
Yes, but I think the parser is still non-validating so it won't really matter. I suspect many of our client xml files are bad. The Java server and client will validated and fail of the namespace and location are bad.